source/geometry/include/TG4GeometryManager.h

```diff
@@ -60,6 +60,7 @@ class TG4GeometryManager : public TG4Verbose
   TG4ModelConfigurationManager* GetEmModelsManager() const;
 
   // functions for building geometry
+  void SetUseExternalGeometryConstruction(G4bool value);
```

Comment:
If this method stays (?), see comments in .cxx,  please move this line after 
  void SetIsMaxStepInLowDensityMaterials(G4bool isMaxStep);